### ISSUE TITLE
Cookie banner focus is unstyled

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk_publishing_components#1014]
I'm not sure if this is intentional but raising it just incase it is not.

When submitting the cookie banner it manages focus to the banner but the focus outline is using a default style.

<img width="1005" alt="" src="https://user-images.githubusercontent.com/2445413/62224538-cad2ba00-b3ae-11e9-855c-2e1bc9f8dbc1.png">

I think... because you're setting focus to have this message read out, and it's not focussable you could hide the outline here...?

```css
.gem-c-cookie-banner__confirmation:focus {
    outline: none;
}
```

Or maybe it should have a consistent outline, not sure :P

**Comment by injms** [comment]
As a part of this, the 'Hide' button focus state should be updated:

<img width="1009" alt="Screen Shot 2019-09-06 at 10 59 27" src="https://user-images.githubusercontent.com/1732331/64419723-739ec400-d095-11e9-8c9f-83ba4344dd55.png">
